src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -933,8 +933,9 @@ Array [
         hello: serialized,
       });
 
-      // TODO: assert that this is expect.any(AlgoliaSearchHelper), but test fails
-      // even though it's an object with all the right properties (including constructor)
+      expect(instantsearch.mainIndex.getHelper()).toEqual(
+        expect.any(AlgoliaSearchHelper)
+      );
```

Comment:
As I was going through the tests (I initially had issues before I got concurrent versions of `algoliasearch-helper`, I saw this `TODO` and I found that it could be solved in a similar manner as in https://github.com/algolia/vue-instantsearch/blob/master/src/util/__tests__/createServerRootMixin.test.js#L970-L975.

Reply:
Sure this makes sense, just felt unrelated to the PR, but it's a small change so I guess it's fine :)